docs/workshop/examples/31_servers_lt_ws/docker-compose.yaml

```diff
@@ -8,7 +8,7 @@ services:
       - 8139:8139
     environment:
       - RUST_BACKTRACE=full
-      - TREMOR_PATH=/opt/local/tremor/lib:/etc/tremor/lib
+      - TREMOR_PATH=/etc/tremor/lib:/usr/local/share/tremor:/usr/share/tremor/lib
```

Comment:
should we remove `/etc/tremor/lib` if we no longer use it? People might put something there -.-

Reply:
some examples already have some necessary code there.

Reply:
should we perhaps update the examples rather than carrying around the baggage? If we have "wrong" code in the examples it will give people bad habits.